doc/tutorials/rt_linux.rst

```diff
@@ -53,17 +53,50 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
 
    a. Download Linux kernel real-time patch::
 
-         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.15-rt12.patch.xz
+         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.31-rt18.patch.xz
 
-   #. Sync the kernel code to acrn-2019w04.5-150000p::
+   #. Sync the kernel code to acrn-2019w17.4-160000p::
 
          $ git clone https://github.com/projectacrn/acrn-kernel.git
-         $ git checkout acrn-2019w04.5-150000p
+         $ git checkout acrn-2019w17.4-160000p
          $ cd acrn-kernel
-         $ xzcat ../patch-4.19.15-rt12.patch.xz | patch -p1
+         $ vim ./mm/swap.c
```

Comment:
Should the user edit the `swap.c` file **after** patching the kernel sources or before? Assuming this is done after, please move this line after the one below (`xzcat`)

Reply:
Several places to resolve if apply patch first, so I prefer to change the original swap.c first, then applying patch will have no any conflicts.

Reply:
In this case I would move the `xzcat` instruction after the patch, so the sequence is not broken up.

Reply:
Followed dbkinder's suggestion by adding a patch to make the steps clear.